packages/beacon-node/src/chain/initState.ts

```diff
@@ -118,7 +118,7 @@ export async function initStateFromEth1({
 
     return genesisResult.state;
   } catch (e) {
-    if (builder.lastProcessedBlockNumber != null) {
+    if (builder.lastProcessedBlockNumber !== null) {
```

Comment:
type of `builder.lastProcessedBlockNumber` is `number | null`. It is safe to do a strict check against `null` as the compiler ensures `builder.lastProcessedBlockNumber` won't be undefined